site/plugins/site/helpers/banner.php

```diff
@@ -51,7 +52,11 @@ function banner(): ?Obj {
     // If a banner is currently active, the cache
     // will also expire when the active banner ends
     if ($banner !== null && $banner->endDate()) {
-        $expires = min($expires, $banner->endDate());
+        if (is_int($expires) === true) {
+            $expires = min($expires, $banner->endDate());
+        } else {
+            $expires = $banner->endDate();
+        }
     }
 
     kirby()->response()->cacheExpiry($expires);
```

Comment:
Do we need to check here if `$expires` has a value?

Reply:
I guess no need. `$expires` default value is `null`, and `cacheExpiry()` accepts `null`.

Reply:
If we pass `null`, the method behaves like a getter, so it will work as well.

Reply:
Ok 👍